recipes/snappy/all/conanfile.py

```diff
@@ -75,10 +81,16 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "Snappy")
+        self.cpp_info.set_property("cmake_target_name", "Snappy::snappy")
+        # TODO: back to global scope in conan v2 once cmake_find_package* generators removed
+        self.cpp_info.components["snappylib"].libs = tools.collect_libs(self)
+        if not self.options.shared and tools.stdcpp_library(self):
```

Comment:
We have problems because of this line, why do we need it? We are trying to use static linking with stdcpp

Reply:
@sunsx0 please, open an issue with more details about your problem. The bug template has a good form of detail which help us to understand your scenario and reproduce your error.